packages/compass-e2e-tests/helpers/commands/click-visible.ts

```diff
@@ -2,11 +2,30 @@ import type { CompassBrowser } from '../compass-browser';
 
 export async function clickVisible(
   browser: CompassBrowser,
-  selector: string
+  selector: string,
+  scrollIntoView = false
 ): Promise<void> {
   const element = await browser.$(selector);
-  await element.scrollIntoView();
+
+  // Allow opting out of scrolling the item into view because that can be quite
+  // finicky for things that only display on hover.
+  if (scrollIntoView) {
```

Comment:
Maybe just to make sure I’m really not missing anything – this seems to be opt-in rather than opt-out?

Reply:
aaargh. default's supposed to be false. Well spotted ;)